### PR TITLE
Cql4bonnie 1086 update

### PR DESCRIPTION
**Pull request by edeyoung**
This pull request addresses external tickets BONNIE-378 and BONNIE-385. There was a bug where "episode of care" was being lost on update. This addresses that issue by making the call to load the measure occur after the measure details are defined.
Additionally, this PR re-adds tests that were removed and makes them work with CQL measures.

Pull requests into Bonnie require the following. (Reviewer should :heavy_check_mark:*when done*):
- [x] This pull request describes *why* these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1086
- [x] JIRA ticket links to this PR.
- [x] JIRA test for this PR: n/a. Appropriately tested in automated tests
- [x] JIRA test has been added to sprint.
   n/a
- [x] Automated regression test(s) pass (if possible): 
- [ ] Regression test screenshots before/after:

### CODE REVIEW

**Review by mayerm94 — DISMISSED** [review]
Changes look good, especially the tests. I ran the regression tests and everything came back good.

File: test/functional/measures_controller_test.rb [hunk 1]
```diff
@@ -176,7 +176,28 @@ class MeasuresControllerTest  < ActionController::TestCase
...
+  test "measure value sets" do
+    sign_in @user
+    measure_file = fixture_file_upload(File.join('test','fixtures', 'cql_measure_exports', 'IETCQL_v5_0_Artifacts.zip'), 'application/xml')
+    class << measure_file
+      attr_reader :tempfile
+    end
+
+    measure = nil
+    VCR.use_cassette("valid_vsac_response") do
+      # Assert measure is not yet loaded
```
Comment (c-monkey) [inline test/functional/measures_controller_test.rb]: This comment appears to be left over from a copy-and-paste from test "upload measure already loaded"

File: test/functional/measures_controller_test.rb [hunk 2]
```diff
@@ -311,11 +332,150 @@ class << update_measure_file
...
+  test "load HQMF bad xml" do
+    sign_in @user
+    measure_file = fixture_file_upload(File.join('test','fixtures', 'cql_measure_exports', 'IETCQL_v5_0_bad_hqmf_Artifacts.zip'), 'application/xml')
+    class << measure_file
+      attr_reader :tempfile
+    end
+
+    measure = nil
+    VCR.use_cassette("valid_vsac_response") do
+      # Assert measure is not yet loaded
```
Comment (c-monkey) [inline test/functional/measures_controller_test.rb]: This comment appears to be left over from a copy-and-paste from test "upload measure already loaded"

**Review by c-monkey — DISMISSED** [review]
Verified re-creation of tests: “measure value sets”, "upload missing value set missing oid”, "create/finalize/update a measure”, “load HQMF bad xml”, and "update a patient based measure"

Verified fixed https://oncprojectracking.healthit.gov/support/browse/BONNIE-385 and https://oncprojectracking.healthit.gov/support/browse/BONNIE-378

Code changes look good. I noted a couple of minor deletions but otherwise approve. If you want to remove these spurious comments, will re-approve.

**Review by c-monkey — APPROVED** [review]
Re-tested, looks good.

**Pull request merged**